person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age:, name: 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @age || parent_permission
```

Comment:
- [ ] Let's use the `of_age?` method and `@parent_permission` instance variable.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```